tests/app/va/vetext/test_client.py

```diff
@@ -163,25 +135,224 @@ def test_increments_statsd_and_timing_on_http_exception(self, rmock, test_vetext
     def test_raises_bad_request_exception_with_info_on_400_error(self, rmock, test_vetext_client, response):
         rmock.post(url=f'{MOCK_VETEXT_URL}/mobile/push/send', status_code=400, json=response)
 
-        with pytest.raises(VETextBadRequestException) as e:
+        with pytest.raises(NonRetryableException):
             test_vetext_client.send_push_notification(
-                'app_sid',
-                'template_sid',
-                'icn',
+                {
+                    'appSid': 1111,
+                    'templateSid': 2222,
+                    'icn': 3333,
+                }
             )
-        assert e.value.field == response.get('idType')
-        assert e.value.message == response.get('error')
 
     def test_raises_bad_request_exception_when_400_error_not_json(self, rmock, test_vetext_client):
         response = 'Unrecognized field &quot;foo&quot; (class gov.va.med.lom.vetext.model.dto.PushNotification),'
         ' not marked as ignorable'
         rmock.post(url=f'{MOCK_VETEXT_URL}/mobile/push/send', status_code=400, text=response)
 
-        with pytest.raises(VETextBadRequestException) as e:
+        with pytest.raises(NonRetryableException):
             test_vetext_client.send_push_notification(
-                'app_sid',
-                'template_sid',
-                'icn',
+                {
+                    'appSid': 1111,
+                    'templateSid': 2222,
+                    'icn': 3333,
+                }
             )
-        assert e.value.field is None
-        assert e.value.message == response
+
+
+class TestFormatVetextPayload:
```

Comment:
see comment above about fixing this longstanding typo `DEAFULT_MOBILE_APP_TYPE` in the many instances seen in this class

Reply:
Updated.